spec/spec_helper.rb

```diff
@@ -20,6 +20,7 @@
   before("extend generic_logger") { Lamian.extend_logger(generic_logger) }
 
   before("stub formatter") do
+    Thread.current[:__lamian_logger] = nil
```

Comment:
How did it work before?

Reply:
I commented it out and specs passed.

Reply:
That's because specs order is not random. But if you change the order it will fail.

Reply:
> That's because specs order is not random. But if you change the order it will fail.

Can you please write exact scenario?

Reply:
If the first example doesn't include `cool_loggers` context, then all other examples will use logger with some default formatter (because of global side effect in `Lamian::Logger.current`).
I added `config.order = :random` now so the specs fail sometimes without this line.

Reply:
Thank you. This behaviour was not intended and i fixed it in #16